frontend/src/pages/phone.page.tsx

```diff
@@ -27,14 +31,19 @@ const Phone: NextPage = () => {
   const realPhoneData = useRealPhonesData();
 
   useEffect(() => {
+    // check if phone flag is active - return to premium page if not.
+    if (isFlagActive(runtimeData.data, "phones")) {
+      Router.push("/premium");
```

Comment:
Oh, also, I'm not sure about lumping this in a single `useEffect` together with unrelated code, because now that code will re-run as well when `runtimeData.data` changes, even though it doesn't do anything with it.

Reply:
Works for me - done.